dbt/adapters/base/relation.py

```diff
@@ -240,6 +279,7 @@ def create_ephemeral_from(
         cls: Type[Self],
         relation_config: RelationConfig,
         limit: Optional[int] = None,
+        event_time_filter: Optional[EventTimeFilter] = None,
```

Comment:
We need to make sure that whatever will be calling this will pin to this version of `dbt-adapters` so that we don't try to pass an extra kwarg to a prior version of `dbt-adapters`.

Reply:
Yes! It will be in this PR: https://github.com/dbt-labs/dbt-core/pull/10594

Will do so once this is released in dbt-adapters.

cc @QMalcolm